gitbook-docs/data_model.md

```diff
@@ -1,104 +1,178 @@
-#Signal K Data Model
+# Signal K Data Model
 
 ## Formats
-Signal K defines two data formats, full and delta, for representing and transmitting data.
 
-In addition the 'sparse'
-format is the same as the full format, but doesn't contain a full tree, just parts of the full tree.
+Signal K defines two data formats—full and delta—for representing and transmitting data. All Signal K data is
+transmitted as UTF-8 JSON.
 
-## Full format
+## Full Format
 
-The simplest format is the full format, which is the complete Signal K data model  as a JSON string.
+The full format is conceptually the simplest representation of data in Signal K. It contains all of the data from a
+Signal K node, which in the case of a Signal K server could me many hundreds of data points.
+
+```json
+{
+  "version": "v1.0.0",
+    "self": "urn:mrn:signalk:uuid:705f5f1a-efaf-44aa-9cb8-a0fd6305567c",
```

Comment:
Self should be relative to root

Reply:
We have #396 open to address this I thought?